go-controller/pkg/ovn/master.go

```diff
@@ -868,12 +868,13 @@ func (oc *DefaultNetworkController) addUpdateRemoteNodeEvent(node *kapi.Node, sy
 		if err := oc.cleanupNodeResources(node.Name); err != nil {
 			return fmt.Errorf("error cleaning up the local resources for the remote node %s, err : %w", node.Name, err)
 		}
+		oc.localZoneNodes.Delete(node.Name)
```

Comment:
ah so this node has now gone remote, we cleanup -> and within cleanupNodeResources we won't err out for not found issues anymore hopefully.. and we will remove from cache and all is well.